samples/net/sockets/packet/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Private config options for packet socket sample app
+
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Networking packet socket sample application"
+
+config NET_SAMPLE_ENABLE_FLOODING
+	bool "Send packets as fast as possible"
+	help
+	  This will stress test the network stack. By default
```

Comment:
The description here is confusing. Does enabling this option causes packets to be sent every 5s? That contradicts the title of the option "Send packets as fast as possible". And if it's not the case, why it's written in that (confusing) way in the description of this option.

Suggested: "If this option not enabled, a packet is sent every 5 seconds". But then, why not add a numeric option, delay in ms, where 0 would mean "flood" ?

Reply:
> The description here is confusing.

What is confusing there. The default value is 'n' which means that a packet is sent every 5 seconds.

Reply:
Allowing user to supply the timeout makes sense, I will implement that.